src/dune_rules/ocaml_stdlib.ml

```diff
@@ -3,7 +3,7 @@ open! Import
 type t =
   { modules_before_stdlib : Module_name.Set.t
   ; exit_module : Module_name.t option
-  ; internal_modules : Glob.t
+  ; internal_modules : Glob.t list
```

Comment:
We usually favor the `Predicate_lang.Glob.t` for things like this.

Reply:
Thanks for the suggestion. Pushed a new version that:

- uses `Predicate_lang.Glob.t`
- versions the `experimental_building_ocaml_compiler_with_dune` syntax to 0.2